ci-operator/config/openshift/loki/openshift-loki-release-5.6.yaml

```
@@ -33,7 +33,7 @@ operator:
   substitutions:
   - pullspec: quay.io/openshift-logging/loki-operator:v0.0.1
     with: pipeline:loki-operator
-  - pullspec: quay.io/openshift-logging/loki:v2.9.3
+  - pullspec: quay.io/openshift-logging/loki:v2.9.4
```

I realized after the lgtm that this won't work ... but I have an idea: What if we always have the "last two pullspecs" in the configuration file? So, in this case that would be 2.9.3 and 2.9.4. That way both the old and new configuration can build. And in the next update we just increment both (e.g. 2.9.4 and 2.9.5).

```suggestion
  - pullspec: quay.io/openshift-logging/loki:v2.9.3
    with: pipeline:loki
  - pullspec: quay.io/openshift-logging/loki:v2.9.4
```